web/wp-content/themes/lfevents/src/assets/scss/modules/_alert.scss

```diff
@@ -0,0 +1,29 @@
+.alert-block  {
+  --bg-color: #ffffff;
+  --text-color: #212326;
+  background-color: var(--bg-color);
+  color: var(--text-color);
+  margin: 0 auto;
+  height: auto;
+  margin-bottom: 1.5rem;
+  max-width: none;
+  width: 100%;
+
+  p {
```

Comment:
Do we need explicit paragraph styles? What if the user adds an `h3`/etc instead?